### PR TITLE
MTDSA-16862: added dynamic GTS to v1 documentation

### PR DESCRIPTION
**Pull request by LouiseWong98**

### CODE REVIEW

**Review by rhysconnor — DISMISSED** [review]
LGTM - will wait to merge!

File: resources/public/api/conf/1.0/retrieve_income_tax_eops.yaml [hunk 1]
```diff
@@ -53,6 +53,20 @@ get:
             <p>Simulate a scenario where no data is found.</p>
           </td>
         </tr>
+        <tr>
+          <td>
+            <p>DYNAMIC</p>
+          </td>
+          <td>
+            <p>Following response values will change to correspond to the values submitted in the request:
```
Comment (ben-HMRC) [inline resources/public/api/conf/1.0/retrieve_income_tax_eops.yaml:61]: "Following" to "The following"

File: resources/public/api/conf/1.0/retrieve_income_tax_income_expenditure.yaml [hunk 2]
```diff
@@ -61,6 +61,20 @@ get:
             <p>Simulate a scenario where no obligations are found.</p>
           </td>
         </tr>
+        <tr>
+          <td>
+            <p>DYNAMIC</p>
+          </td>
+          <td>
+            <p>Following response values will change to correspond to the values submitted in the request:
```
Comment (ben-HMRC) [inline resources/public/api/conf/1.0/retrieve_income_tax_income_expenditure.yaml:69]: "Following" to "The following"

**Review by ben-HMRC — CHANGES_REQUESTED** [review]
See comments on individual lines

**Review by rhysconnor — APPROVED** [review]
LGTM

**Pull request merged**